charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet-config-file

```diff
@@ -34,13 +34,13 @@ eventBurst: 50
 evictionHard:
   imagefs.available: 5%
   imagefs.inodesFree: 5%
-  memory.available: 100Mi
+  memory.available: 1Gi
   nodefs.available: 5%
   nodefs.inodesFree: 5%
 evictionSoft:
   imagefs.available: 10%
   imagefs.inodesFree: 10%
-  memory.available: 200Mi
+  memory.available: 1.5Gi
```

Comment:
I would rather change this to a percentage - this is a hard punishment for a machine with low base memory.

Reply:
i haven't done experiments with concrete numbers, however, picked up these numbers from the examples in documentation. The example talked about 10% or 1GB for a machine with 10GB of memory. Keeping 5% or 10% for the machines of sizes 60GB or 120GB, and reserving so much memory without any experiments doesn't seem very good.
WDYT? Do you have any recommendations on how to proceed with percentages for different sizes of machines, or can we go with the current implementation

Reply:
I'll rather have percentage for a machine with low memory - let's say 10-15gb and after that a static number.

For machine with `< 8Gi` memory (threshold value is just an example) we might put `10%` for soft and `5%` for hard.
For machine with `> 8Gi` memory (threshold value is just an example) we might put `1Gi` for soft and `1.5Gi` for hard.

Reply:
Thanks @mvladev for suggestions. I will submit the change involving eviction thresholds separately in that case. This will require right design such that the memory size can be queried for machines from different cloud provider (possibly different OS...?), and the values can be configured accordingly. Right now the values are static.

Reply:
This information is already available on the CloudProvider https://github.com/gardener/gardener/blob/6d8de051b67b470f7a6eab9102dc961d79fa9741/example/30-cloudprofile-aws.yaml#L35-L39

Reply:
Thanks for the pointer @mvladev. I will raise another PR for the eviction threshold changes